### PR TITLE
Add @bdice and @vyasr as maintainers

### PR DESCRIPTION
**Pull request by gmarkall**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] ~~Bumped the build number (if the version is unchanged)~~
* [ ] ~~Reset the build number to `0` (if the version changed)~~
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Review by bdice — APPROVED** [review]

File: recipe/recipe.yaml [hunk 1]
```diff
@@ -43,4 +43,7 @@ about:
 
 extra:
   recipe-maintainers:
+    - bdice
     - gmarkall
+    - jakirkham
+
```
Comment (bdice) [inline recipe/recipe.yaml:49]: Linter is complaining about this extra line:
```suggestion
```

File: recipe/recipe.yaml [hunk 2]
```diff
@@ -43,4 +43,6 @@ about:
 
 extra:
   recipe-maintainers:
+    - bdice
     - gmarkall
+    - jakirkham
```
Comment (jakirkham) [inline recipe/recipe.yaml:48]: I probably cannot commit significant time to this. So will depend on Graham to drive this forward. If you have questions or run into issues, am happy to chat though

Comment (gmarkall) [reply recipe/recipe.yaml]: Many thanks - as Vyas volunteered, I've switched your user ID for his on this PR.

Comment (jakirkham) [reply recipe/recipe.yaml]: Sounds good. Thanks Graham! 🙏

Am now watching the feedstock. Though probably best to ping me if it is important (lest it get buried by all the other conda-forge notifications 😅)

**Pull request merged**